stock_picking_auto_create_lot/models/stock_move_line.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    @api.model
```

Comment:
I think it's more appropriate not to have the decorator. We expect to work on a given record when it's extended.
```suggestion
```